### PR TITLE
Fix delete post/comment modal UI

### LINKED ISSUE #1929: Fix delete post modal UI
Should be [-> this <-](https://www.figma.com/file/EMKcg0eTnOlr7bRZGFdnZu/Web-app-product-spec?node-id=133%3A14)
Current:
image

### PR DESCRIPTION
**Pull request by Zaydme**
### What does this pull request aim to achieve? 💯
Styling the post/comment delete modal,
turns this
image
into this
image
Following this design:
image

_Please be concise and link any related issue(s):_
closes #1929

- [x] I have checked that no one else is working on similar changes.
- [x] I have read the latest [**README**](README.md) and understand the development workflow.
- [x] The name of this branch is: **`feature/<branch_name>`**, or **`hotfix/<branch_name>`** if this is a hotfix, where `<branch_name>` briefly describes the issue(s) resolved and is prefixed with the issue number(s) (e.g. `1127-1130-update-git-branching-model`). The branch is created in a cloned version of this repo, **not a forked repo**.
- [x] This branch is rebased/merged with the **latest staging** (or the **latest production** if it is a hotfix).
- [x] There are no merge conflicts.
- [x] There are no console warnings and errors.
- [x] On the right hand side of this PR, I have linked the appropriate issue(s) under "Issues" and added a project progress under "Projects".
- [x] The title describes the issue(s) being addressed using format: **`<issue_numbers> - <brief_issue_description>`** (e.g. `#1127/#1130 - Update Git Branching Model`)

### CODE REVIEW

**Comment by rabbi08** [comment]
@Zaydme 
In the text 'Are you sure you want to delete the post?"
It should be 'THIS' not 'THE' as per the design. Please update as 

Are you sure you want to delete this post?

File: client/src/components/Feed/PostDeleteModal.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+                font-size: ${large};
+                height: 100%;
+                padding: 0.8rem 2.2rem;
+                border-radius: 46px;
+                &:first-child {
+                    color: ${royalBlue};
+                }
+                &:last-child {
+                    color: white;
+                    background-color: #FF5656;
```
Comment (ddveloper) [inline client/src/components/Feed/PostDeleteModal.js:43]: maybe we can create a separate issue to collect all colors into the constant file, low priority

**Review by ddveloper — APPROVED** [review]

File: client/src/components/Feed/PostDeleteModal.js (hunk 1 above)
Comment (Zaydme) [reply client/src/components/Feed/PostDeleteModal.js:43]: Yep sounds good

**Comment by rabbi08** [comment]
Verified the fix in feature build. It's working as expected. Refer below screenshot.

image

**Review by rabbi08 — APPROVED** [review]

**Review by CamilaArias — APPROVED** [review]
lgtm

**Pull request merged**